pkg/flag/cache_flags.go

```diff
@@ -109,14 +111,13 @@ func (fg *CacheFlagGroup) ToOptions() (CacheOptions, error) {
 		return CacheOptions{}, err
 	}
 
-	backendOpts, err := cache.NewOptions(fg.CacheBackend.Value(), fg.RedisCACert.Value(), fg.RedisCert.Value(),
-		fg.RedisKey.Value(), fg.RedisTLS.Value(), fg.CacheTTL.Value())
-	if err != nil {
-		return CacheOptions{}, xerrors.Errorf("failed to initialize cache options: %w", err)
-	}
-
 	return CacheOptions{
-		ClearCache:          fg.ClearCache.Value(),
-		CacheBackendOptions: backendOpts,
+		ClearCache:   fg.ClearCache.Value(),
```

Comment:
IIUC we don't have to do this.
`ClearCache` is no longer used.
We also return error for this flag:
```bash
➜ ./trivy -d image --clear-cache
2024-06-27T11:30:39+06:00       DEBUG   Cache dir       dir="/home/dmitriy/.cache/trivy"
2024-06-27T11:30:39+06:00       ERROR   "--clear-cache" was removed. Use "trivy clean --scan-cache" instead
2024-06-27T11:30:39+06:00       FATAL   Fatal error     flag error: cache flag error: unable to parse flag: removed flag ("--clear-cache")

```

Reply:
Removed https://github.com/aquasecurity/trivy/pull/7024/commits/194cef93078b8a7d55ccb0b2c50237dfa68aa190